tools/fonts/inspector/glyphs.html

```diff
@@ -96,54 +159,65 @@ <h1 id="title">Glyph Inspector</h1>
         ctx.setTransform(2, 0, 0, 2, 0, 0); // reset the scale to (2, 2)
 
         var x = 0;
-        var y = 50;
-        for (let glyphName in glyphs) {
+        var y = 250;
+
+        const arr = Object.keys(SMuFLGlyphInfo).sort(function(a,b) { 
+          return parseInt(SMuFLGlyphInfo[a].codepoint.substring(2), 16) - parseInt(SMuFLGlyphInfo[b].codepoint.substring(2), 16); 
+        } );
```

Comment:
Can you add a good comment here explaining this string manipulation?

I know what it is doing since we need to translate `"codepoint": "U+E30C",` to something you can display. However, if I read this code a year from now, I'll be confused. :-)

Reply:
Moved to a function with comment